src/parser/jobs/nin/NinCombos.js

```diff
@@ -0,0 +1,13 @@
+import ACTIONS from 'data/ACTIONS'
+import Combos from 'parser/core/modules/Combos'
+
+export default class NinCombos extends Combos {
```

Comment:
Preference renaming the parent at import time. Instead of `nin/NinCombos`, you can have `nin/Combos` by doing:

```js
import CoreCombos from 'parser/core/modules/Combos'
export default class Combos Extends CoreCombos {
}
```

Oh, and the parent class already has the handle, no need to re-declare.

Reply:
Noted, will update.